internal/connector/internal/handlers/connector_cluster.go

```diff
@@ -39,6 +39,7 @@ type ConnectorClusterHandler struct {
 	Keycloak           sso.KafkaKeycloakService
 	Connectors         services.ConnectorsService
 	ConnectorNamespace services.ConnectorNamespaceService
+	Processors         services.ProcessorsService
```

Comment:
`connector_cluster.go` contains functions relating to _resources_ within a `connector_cluster`.

A `connector_cluster` can now also include `processor`'s so some of the methods in this file relate to them.

Reply:
Arguably the term `connector_cluster` is now misleading.. but I've resisted temptation to refactor it!